Source/JavaScriptCore/runtime/ProgramExecutable.cpp

```diff
@@ -134,7 +134,7 @@ JSObject* ProgramExecutable::initializeGlobalProperties(VM& vm, JSGlobalObject*
                     if (globalLexicalEnvironment->isConstVariable(entry.key.get()))
                         continue;
                 }
-                return createSyntaxError(globalObject, makeString("Can't create duplicate variable: '", String(entry.key.get()), "'"));
+                return createSyntaxError(globalObject, makeString("Can't create duplicate variable: '"_s, String(entry.key.get()), '\''));
```

Comment:
Should be able to pass `entry.key` without `String` to avoid reference count churn.